### PR TITLE
Ana/fix claim rewards details styles and add all rewards

### PR DESCRIPTION
**Pull request by Bitcoinera**
Only works together with luniehq/lunie-api#493

**Description:**

Making the ClaimRewardsTxDetails great again!

Inspired but not strictly following Charlie's design (since some things don't fit how currently the details components are displayed).

This is how it looks now:

- For multiple validators:

image

image

- For a single validator:

image

(Single validator open: this is the case I am most doubtful about)

image

Please, do share all your suggestions & design wisdom & awesome CSS skills 🙏 

Thank you! 🚀

---

For contributor:

- [ ] Added changes entries. Run `yarn changelog` for a guided process.
- [ ] Reviewed `Files changed` in the github PR explorer
- [ ] Attach screenshots of the UI components on the PR description (if applicable)
- [ ] Scope of work approved for big PRs

For reviewer:

- [ ] Manually tested the changes on the UI

### CODE REVIEW

**Comment by jbibla** [comment]
the top row should always be top-aligned:

Screen Shot 2020-03-21 at 3 11 33 PM

**Comment by Bitcoinera** [comment]
I just applied some changes (and broke couple of things along the day). 

Tomorrow will be another day 😆

**Comment by Bitcoinera** [comment]
OK, it has been challenging (with so many possible situations), but I'd call this now ready! 💪 

All suggestions have been applied ✌️ 

Addresses to check how it looks like:

- Multi-validators with one single denom: 
cosmos1rfg85mh5pmm5f0m05wmqg5qaknv3u7rfyqp36t (gaia testnet)
- One single validator and one denom: cosmos1u6d6usvgz8hjzd7dwgpfegk4xf9j83frc7wunf (gaia testnet) (you may not be able to look at this...)
- Both multi and single validators with multiple denoms: emoney1yw2xd42y79t650n55ddp6pjdp9sd97syz03ql9 (emoney-testnet)

File: src/components/transactions/message-view/ClaimRewardsTxDetails.vue [hunk 1]
```diff
@@ -341,7 +341,7 @@ export default {
 .reponsiveClaimValidator {
   margin: 0 2rem;
 }
-@media screen and (max-width: 1035px) {
+@media screen and (max-width: 822px) {
```
Comment (faboweb) [inline src/components/transactions/message-view/ClaimRewardsTxDetails.vue]: @jbibla this seems hacky. shouldn't breakpoints be standardized to just a couple?

Comment (jbibla) [reply src/components/transactions/message-view/ClaimRewardsTxDetails.vue]: yes

Comment (Bitcoinera) [reply src/components/transactions/message-view/ClaimRewardsTxDetails.vue]: Thank you for the remark. This code was actually not needed and just deleted it.

Let's get luniehq\lunie-api#493 and this merged! 🚀 💃

**Comment by Bitcoinera** [comment]
Amazing! Now we can also release extension 🚀 !!

**Pull request merged**